### DISCUSSION TITLE
first layer temp

### DISCUSSION THREAD

**Discussion by PVerburg** [supermerill/SuperSlicer#3617] [Q&A]
observation first-layer temp not used on 0.28 layer height when slicing it showes right temp at 0.20 but never used as first layer is on 0.28 anybody else seen this behavior ??

**Comment by neophyl** [comment]
No, not seen this.  Just tried it with a part in 2.5.59 and its not doing it for me.

**Comment by PVerburg** [comment]
Thanks I’m on 2.4.58.5 I should upgrade and try again.
